lib/sensu-handler.rb

```diff
@@ -209,6 +209,7 @@ def filter_dependencies
         if @event['check']['dependencies'].is_a?(Array)
           @event['check']['dependencies'].each do |dependency|
             begin
+              next if dependency.empty?
```

Comment:
@rs-mrichmond On second thought, perhaps we should consider converting `dependency` to a string first? `next if dependency.to_s.empty?` would protect us from undefined method errors if the dependency were a Fixnum or NilClass object. cc @fessyfoo